### PR TITLE
[improve][e2e] Change the starter module from file copy to binding file path

### PR DESCRIPTION
**Pull request by ashulin**
## Purpose of this pull request
The start module file can be determined before the container is started and don‘t need to be copied.

## Check list

* [x] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

**Review by Hisoka-X — DISMISSED** [review]
LGTM

File: seatunnel-e2e/seatunnel-e2e-common/src/test/java/org/apache/seatunnel/e2e/common/container/seatunnel/SeaTunnelContainer.java [hunk 1]
```diff
@@ -54,13 +52,15 @@ public class SeaTunnelContainer extends AbstractTestContainer {
     public void startUp() throws Exception {
         server = new GenericContainer<>(getDockerImage())
             .withNetwork(NETWORK)
-            .withCommand(Paths.get(SEATUNNEL_BIN, SERVER_SHELL).toString())
+            .withCommand(Paths.get(SEATUNNEL_HOME, "bin", SERVER_SHELL).toString())
             .withNetworkAliases("server")
             .withExposedPorts()
-            .withLogConsumer(new Slf4jLogConsumer(LOG))
+            .withLogConsumer(new Slf4jLogConsumer(DockerLoggerFactory.getLogger("seatunnel-engine:" + LOG)))
```
Comment (hailin0) [inline seatunnel-e2e/seatunnel-e2e-common/src/test/java/org/apache/seatunnel/e2e/common/container/seatunnel/SeaTunnelContainer.java]:
```suggestion
            .withLogConsumer(new Slf4jLogConsumer(DockerLoggerFactory.getLogger("seatunnel-engine:" + JDK_DOCKER_IMAGE)))
```

**Review by hailin0 — COMMENTED** [review]
LGTM

**Review by EricJoy2048 — DISMISSED** [review]
LGTM

**Comment by EricJoy2048** [comment]
Resolve conflicts please.

**Comment by ashulin** [comment]
@EricJoy2048 @Hisoka-X PTAL again.

**Review by EricJoy2048 — APPROVED** [review]
LGTM

**Review by TyrantLucifer — APPROVED** [review]
LGTM

**Comment by TaoZex** [comment]
LGTM

**Review by TaoZex — APPROVED** [review]

**Pull request merged**